src/backfill_unpublished.py

```diff
@@ -0,0 +1,28 @@
+import adaptor, fs_adaptor, utils
+import json
+
+# not working, untested
+
+def main():
+    cmd = [
+        adaptor.find_lax(), # /srv/lax/manage.sh
+        "status",
+        "article-versions.invalid-unpublished.list"
+    ]
+    lax_stdout = None
+    rc, lax_stdout = utils.run_script(cmd)
+
+    requests = []
+    for av in json.loads(lax_stdout)['article-versions.invalid-unpublished.list']:
```

Comment:
the `article-versions.invalid-unpublished.list` field access looks a bit strange, what other fields are there?